java-vtl-script/src/test/java/no/ssb/vtl/script/VTLScriptEngineAdvancedTest.java

```diff
@@ -41,6 +46,63 @@
     private ScriptEngine engine = new VTLScriptEngine(connector);
     private Bindings bindings = engine.getBindings(ScriptContext.ENGINE_SCOPE);
 
+    @Test
+    public void testManyOrsBools() throws Exception {
+        runExpressionLoop("true","or");
+    }
+
+    @Test
+    public void testManyAndBools() throws Exception {
+        runExpressionLoop("true", "or");
```

Comment:
Typo? Shouldn't it be `runExpressionLoop("true", "and");`

Reply:
Yes. Thank you. I have fixed.